Header header
float32 inchange_ch0
float32 vel_ch0
float32 cur_ch0
float32 inchange_ch1
float32 vel_ch1
float32 cur_ch1

